syntax = "proto3";


package todoPackage;

service Todo{
	rpc registrarVisitante(RegVis) returns(RegReturns){}
	rpc editarVisitante(EditVis) returns(RegReturns){}
}

message RegReturns{
	string id = 1;
	string name = 2;
	bool ok= 3;
	string msg = 4;
}

message RegVis{
	string name = 1;
	string password = 2;
}

message EditVis{
	string name = 1;
	string password = 2;
	string newName = 3;
	string newPassword = 4;
}